<launch>
  <node pkg="skill_transfer" type="knowledge_manager" name="knowledge_manager" output="screen">
    <param name="task_file_path" type="string" value="$(find skill_transfer)/tasks/$(arg task).yaml" />
    <param name="motion_template_file_path" type="string" value="$(find skill_transfer)/motion_templates/$(arg robot).yaml"/>
    <param name="motion_directory_path" type="string" value="$(find skill_transfer)/motions/" />
    <param name="setup_file_path" type="string" value="$(find skill_transfer)/setups/$(arg setup).yaml" />
  </node>

  <node pkg="skill_transfer" type="constraint_controller_$(arg robot)" name="constraint_controller" output="screen"/>

  <node pkg="skill_transfer" type="task_executive2" name="task_executive" output="screen"/>
</launch>